2023/sg-first-year-candidates/main.js

```diff
@@ -0,0 +1,81 @@
+// For API and chart documentation please look here:
+// https://www.highcharts.com/demo
+Highcharts.chart('chart-div', {
+    // Setting default colors
+    colors: [
+        '#800000',
+        '#FFA319',
+        '#C16622',
+        '#8F3931',
+        '#8A9045',
+        '#58593F',
+        '#155F83',
+        '#350E20',
+        '#47B5FF',
+        '#FF3399'
+    ],
+
+    // All code for your chart goes here
+    title: {
+        text: 'First-Year Candidates for College Council by Year',
+        align: 'left'
```

Comment:
Titles are centered. I also think `First-Year Candidates Running for College Council: 2016 to 2023` would be a better title.